connect/api/src/test/java/org/apache/kafka/connect/connector/ConnectorReconfigurationTest.java

```diff
@@ -78,5 +79,11 @@ public void stop() {
             if (stopException)
                 throw new ConnectException("error");
         }
+
+        @Override
+        public List<ConfigValue> validate(Map<String, String> connectorConfigs) {
+            return null;
```

Comment:
Is null a valid return value? If so, we should probably document that and make sure the code that calls this method handles null properly.